charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -190,6 +190,20 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - roles
+  - rolebindings
+  verbs:
+  - create
+  - delete
+  - deletecollection
```

Comment:
This will allow the RoleBindings `gardener.cloud:system:project-member` and `gardener.cloud:system:project-viewer` to be deleted (until they are later reconciled), right? Do we want this?

Reply:
Yes, it'll allow deletion of those rolebindings. Is it correct - no. Do we want this - no, but this is why we have reconciliation ?

Reply:
> Yes, it'll allow deletion of those rolebindings. Is it correct - no. Do we want this - no, but this is why we have reconciliation ?

Afaik we do not guarantee anything about gardener-controller-manager availability.
It sounds to me like a `CVE` to be able to drop the access of project-members, viewers and serviceaccounts at any time.